File: kvadraticka f/druhy.py
import math

a = int(input("Zadaj a"))
b = int(input("Zadaj b"))
c = int(input("Zadaj c"))

print("Tvoja rovnica je :", a,"x*x+",b,"x+",c)

D = b*b - 4 *a *c

if(D < 0):
    print("Rovnica nemá riešenie")
elif(D == 0):
    print("Rovnica má jedno riešenie")
    x1 = -b/2*a
    print("X1 je:",round((x1),2))
else:
    odmocninaD = round( math.sqrt(D),2)
    print("Odmocnina s diskriminantu je:",odmocninaD)
    x1 = (-b-odmocninaD)/2*a
    x2 = (-b+odmocninaD)/2*a
    print("X1 je:",round((x1),2)," X2 je: ", round((x2),2))

File: kvadraticka f/druhy.c
#include<stdio.h>
#include<math.h>

int main(){
	int a, b, c;
	float odmocninaD, x1, x2, D;
	
	printf("Zadaj a: ");
	scanf("%d" ,  &a);
	printf("Zadaj b: ");
	scanf("%d" ,  &b);
	printf("Zadaj c: ");
	scanf("%d" ,  &c);
	
	printf("Tvoj tvar rovnice je: %dx*x + %dx + %d",a,b,c);
	
	D = b*b - 4 *a *c;
	printf(" \n %f",D);
	
	    
	if(D<0){
		printf("Rovnica nema riesenie.");
	}else if(D>0){
	    x1 = (-b- round(sqrt(D))) /2*a;
	    x2 = (-b+ round(sqrt(D))) /2*a;
	    printf("X1 je: %f X2 je: %f",x1,x2);
	}else{
		printf("Rovnica ma jedno riesenie.");
		x1 = -(b)/(2*a);
   		printf("X1 je: %f",x1);
	}


	
}
